actionpack/test/dispatch/show_exceptions_test.rb

```diff
@@ -98,7 +98,7 @@ def call(env)
 
   test "calls custom exceptions app" do
     exceptions_app = lambda do |env|
-      assert_kind_of AbstractController::ActionNotFound, env["action_dispatch.exception"]
+      assert_kind_of ActionView::Template::Error, env["action_dispatch.exception"]
```

Comment:
#18774 didn't change this assertion; did it really previously contain the outermost exception? :confused: